README.md

```diff
@@ -57,7 +57,7 @@ apt install grub-btrfs
 ```
 Booting into read-only snapshots is fully supported when choosing btrfs as the file system during a standard Kali Linux installation following [this walk-through](https://www.kali.org/docs/installation/btrfs/).
 
-#### Manual
+#### Manual installation from source
```

Comment:
Lets make this into "Manual installation" or "Installation from GitHub" instead. 
"from source" is always a bit confusing for sofware that consists entirely of  a scripting language

Reply:
Scripts still count as source code but OK.